components/list/list-item-drag-mixin.js

```diff
@@ -1,20 +1,77 @@
+import { css, html } from 'lit-element/lit-element.js';
 import { announce } from '../../helpers/announce.js';
+import { dragActions } from './list-item-drag-handle.js';
 import { getUniqueId } from '../../helpers/uniqueId.js';
-import { html } from 'lit-element/lit-element.js';
 import { nothing } from 'lit-html';
 
+const move = Object.freeze({
+	above: true,
+	below: false
+});
+
+const timeDelayForLeavingTheDropArea = 1000; //ms
```

Comment:
This is to give drag n drop more usability. Basically, if you go past the drop point ie top of the list it will still work for 1 second. I found this to be a better solution then trying to increase the drop target size.